cell_classification/segmentation_data_prep.py

```diff
@@ -118,16 +128,18 @@ def marker_activity_mask(self, instance_mask, cell_types, marker_activity):
         pass
 
     def make_tf_record(self, tf_record_path):
-        """ Iterates through the data_folders and loads, transforms and serializes a
+        """Iterates through the data_folders and loads, transforms and serializes a
```

Comment:
Let's flesh out this function with the logic of how the helper functions are being called to make sure the parts that need to work across multiple cells, and multiple markers, will play well with each other